internal/pkg/cli/deploy/env.go

```diff
@@ -100,11 +141,115 @@ func NewEnvDeployer(in *NewEnvDeployerInput) (*envDeployer, error) {
 			Env:             in.Env,
 		},
 		newStackSerializer: func(in *deploy.CreateEnvironmentInput, lastForceUpdateID string, oldParams []*awscfn.Parameter) stackSerializer {
-			return stack.NewEnvConfigFromExistingStack(in, lastForceUpdateID, oldParams)
+			return cfnstack.NewEnvConfigFromExistingStack(in, lastForceUpdateID, oldParams)
+		},
+		envDescriber:      envDescriber,
+		envManagerSession: envManagerSession,
+		newLBDescriber: func(sess *session.Session) lbDescriber {
+			return elbv2.New(sess)
+		},
+		newServiceStackDescriber: func(svc string, sess *session.Session) stackDescriber {
+			return stack.NewStackDescriber(cfnstack.NameForService(in.App.Name, in.Env.Name, svc), sess)
```

Comment:
do we need to accept a session as a parameter? if I'm following correctly the session seems to be always the `envManagerSession`
```suggestion
		elb: elbv2.New(envManagerSession)
		newServiceStackDescriber: func(svc string) stackDescriber {
			return stack.NewStackDescriber(cfnstack.NameForService(in.App.Name, in.Env.Name, svc), envManagerSession)
```